payments-core/src/main/java/com/stripe/android/view/CardInputWidget.kt

```diff
@@ -699,6 +700,10 @@ class CardInputWidget @JvmOverloads constructor(
                 R.styleable.CardInputView_android_focusedByDefault,
                 true
             )
+            imeLandscapeFullScreen = getBoolean(
+                R.styleable.CardInputView_imeLandscapeFullScreen,
+                true
```

Comment:
Do we need to be setting this programmatically at all? Does setting ```android:imeOptions="flagNoFullscreen"``` 

break behavior elsewhere? We could get rid of all of this logic right for the xml tag right?

Reply:
It's ok. 👍 

There are some `imeOptions` in Android.  e.g. `actionSearch`.
But only `flagNoFullScree` works.
Is that okay with you?

Reply:
If we're just needing the single flag and it doesn't break the page outside of landscape I think we should be doing it in XML. If we need other flags we can add them later programmatically as needed.